backend/model/User.py

```diff
@@ -0,0 +1,79 @@
+from errors import BadRequest, UserAlreadyExistsError, UserNotFoundError
+
+
+class User:
+    def __init__(
+        self,
+        account_id: int = None,
```

Comment:
I think it's a good idea to keep nullable values consistent with the schema. For example here `account_id`, `username`, `email`, and `password` shouldn't be nullable.

This should especially be the case for `id`. `None` shouldn't be allowed for `id` in any case.

Reply:
`None` is allowed for `id` when creating a user to add to the database because I thought the database had some auto-incrementing id?